packages/forklift-console-plugin/src/modules/Plans/utils/helpers/getPlanSummaryStatus.ts

```diff
@@ -0,0 +1,74 @@
+import { PlanData, PlanSummaryStatus } from '../types';
+import {
+  PlanConditionCategory,
+  PlanConditionStatus,
+  PlanConditionType,
+} from '../types/PlanCondition';
+
+export const getPlanSummaryStatus = (data: PlanData): PlanSummaryStatus => {
+  const plan = data?.obj;
+  const conditionTypes = plan?.status?.conditions?.reduce((acc, condition) => {
+    if (condition.status === PlanConditionStatus.True) {
+      acc.push(condition.type);
+    }
+
+    return acc;
+  }, []);
+
+  if (!conditionTypes?.length) {
+    return;
+  }
+
+  const isArchiving = plan?.spec?.archived && !conditionTypes.includes(PlanConditionType.Archived);
+
+  // Archived
+  if (isArchiving || conditionTypes.includes(PlanConditionType.Archived)) {
```

Comment:
i dont understand this conidtion, seems lik it true when 
plan?.spec?.archived && !conditionTypes.includes(PlanConditionType.Archived);
or
 conditionTypes.includes(PlanConditionType.Archived) and u don't care about plan?.spec?.archived ?

Reply:
I believe this is timing related. The logic was borrowed from the existing `getPlanPhase`, where I think there's a potential for `plan?.spec?.archived` to be marked as true while a condition is yet to exist with a status of "True". I assume the spec value is immediate and the condition takes a few moments to populate. I could be wrong though.

Reply:
So if i still remember something regarding de morgn laws I think we can change it to 
plan?.spec?.archived || conditionTypes.includes(PlanConditionType.Archived)
lets call plan?.spec?.archived = A
conditionTypes.includes(PlanConditionType.Archived) = B
we have (A && !B) || B , so if B is true we pass, if B is false we have result by A , so A || B is just the same

Reply:
That makes sense, yes, that should also work. I can update both this new helper and `getPlanPhase` with this improvement/simplification.